src/controllers/users.controller.ts

```diff
@@ -0,0 +1,71 @@
+import {ValidatedRequest} from 'express-joi-validation';
+import * as userValidation from '../validation/validation-schemas';
+import UserService from '../services/user.service';
+import {Request, Response} from 'express';
+import {RESPONSE_MESSAGES} from '../config/messages';
+
+export class UsersController {
```

Comment:
some API endpoints return passwords, isDeleted
* Create
```json
{
    "isDeleted": false,
    "id": "e9fa79b9-b010-47d8-afa9-9e8f40d1f361",
    "login": "qwe@!",
    "password": "$2b$10$dw/ejL3Rk9OpYmqEIRvVHuzRoKFViIJSlBHPSeqKJT3jXrPm1oe9K",
    "age": 30,
    "updatedAt": "2023-02-06T11:59:26.880Z",
    "createdAt": "2023-02-06T11:59:26.880Z"
}
```
Get by id
```json
{
    "id": "e9fa79b9-b010-47d8-afa9-9e8f40d1f361",
    "login": "qwe@!",
    "age": 30
}
```

and so on.

Please, make sure every response is consistent

Reply:
fixed for every api.
For user the solution is not the most elegant because operator 'returning' doesn't work for sequelize create